Docs/AcpiSamples/SSDT-DMAC.dsl

```diff
@@ -0,0 +1,57 @@
+/*
+ * Only necessary when no DMAC device (with any name) is present in the DSDT and a custom device-id
```

Comment:
Why is that device needed? Also not very descriptive to be honest.

Reply:
the descriptions need to be changed, I was waiting for your comment to change them

Reply:
dmac e fwhd, these devices are missing on dsdt from 6 gen intel onwards often, and on real mac there are.